server/src/main/java/io/pravega/schemaregistry/storage/impl/schemas/PravegaKeyValueSchemas.java

```diff
@@ -114,8 +123,16 @@ public PravegaKeyValueSchemas(TableStore tableStore) {
                 new VersionedRecord<>(new SchemaIdList(schemaIdList).toBytes(), fingerprintKeyVersion));
         // 2. add schemaId record
         SchemaIdKey key = new SchemaIdKey(id);
+        List<ByteArraySegment> chunks = ChunkUtil.chunk(schemaInfo.getSchemaData(), Config.MAX_CHUNK_SIZE_BYTES);
+        assert !chunks.isEmpty();
```

Comment:
While its a good way of asserting and documenting assumptions, it can easily be bypassed in production (if the `-ea` VM option is missing). In this particular case, throwing an `IllegalStateException` may be better.

Reply:
removed the assert. 
we dont intend to check for this state, rather this is expected behaviour from the contract with the ByteBufferChunks library. 
Even for an empty buffer, it will return a single chunk with an empty buffer.